components/yaml.rst

```diff
@@ -102,13 +102,8 @@ string and converts it to a PHP array:
 
     use Symfony\Component\Yaml\Yaml;
 
-    $value = Yaml::parse(file_get_contents('/path/to/file.yml'));
-
-.. caution::
-
-    Because it is currently possible to pass a filename to this method, you
-    must validate the input first. Passing a filename is deprecated in
-    Symfony 2.2, and was removed in Symfony 3.0.
+    $value = Yaml::parse("foo:\n    'bar'");
```

Comment:
having the newline here looks wrong to me, I would just use `$value = Yaml::parse('foo: bar');`